fourth review/swagger.yaml

```diff
@@ -0,0 +1,1366 @@
+        
+            
+    
+    delete:
+      description: Remove a draft order
+      tags:
+        - Delete
+      parameters:
+        - name: id
+          in: path
+          description: draft order id
+          required: true
+          type: integer
+          format: int64
+      responses:
+        '200':
+          description: Deleted draft order
+          schema:
+            $ref: '#/definitions/DraftOrder'
+            
+        '400':
+          description: Bad Request
+          schema:
+            type: object
+            properties:
+              errors:
+                  $ref: '#/definitions/Error'
+                  
+            
+      x-code-samples:
+        - lang: curl
+          source: |
+            curl -X DELETE -H "Content-Type: application/json" https://#{api_key}:#{api_password}@your_store_domain.myshopify.com/admin/draft_orders/#{id}.json
+            
+        - lang: Ruby
+          source: |
+              require 'shopify_api'
+
+              shop_url = "https://#{api_key}:#{api_password}@your_store_domain.myshopify.com/admin"
+              ShopifyAPI::Base.site = shop_url
+              draft_order = ShopifyAPI::DraftOrder.find(#{id})
+              draft_order.destroy    
+      
+  /admin/draft_orders/count.json:
+    get:
+      summary: Return count of draft orders
+      description: This endpoint enables you to return a count of all draft orders.    
+      parameters:
+       - $ref: '#/parameters/fields'
+       - $ref: '#/parameters/limit'
+       - $ref: '#/parameters/page'
+       - $ref: '#/parameters/since_id'
+       - $ref: '#/parameters/status'
+       - $ref: '#/parameters/ids'
+      tags:
+        - Get
+      responses:
+        '200':
+           $ref: '#/responses/Integer'
+           
+        '400':
+          description: Bad Request
+          schema:
+            type: object
+            properties:
+              errors:
+                  $ref: '#/definitions/Error'
+                  
+
+      x-code-samples:
+        - lang: curl
+          source: |
+            curl -X GET -H "Content-Type: application/json" https://#{api_key}:#{api_password}@your_store_domain.myshopify.com/admin/draft_orders/count.json
+      
+        - lang: Ruby
+          source: |
+            require 'shopify_api'
+
+            shop_url = "https://#{api_key}:#{api_password}@your_store_domain.myshopify.com/admin"
+            ShopifyAPI::Base.site = shop_url
+            draft_order = ShopifyAPI::DraftOrder.count
+
+
+
+  '/admin/draft_orders/#{id}/send_invoice.json':
+    post:
+      summary: Send an invoice for a draft order
+      description: You can use this endpoint to send your customer an invoice via email for their draft order.  The invoice contains a url link, enabling the customer to pay for the item using web checkout.  Additionally, you can send a `draft_order_invoice` hash in the body of the request to override any of the default email parameters for the invoice.  
```

Comment:
* "via email" -> "by email"
* url -> URL
* "link, enabling" -> "link that enables"
* "web checkout" -> "a web checkout"
* last sentence - consider inverting this to begin with the condition: "If you want to override any of the default email parameters for the invoice, then you can send a `draft_order_invoice` hash in the body of the request."